src/graphql/validation/rules/single_field_subscriptions.py

```diff
@@ -23,62 +23,63 @@ class SingleFieldSubscriptionsRule(ValidationRule):
     def enter_operation_definition(
         self, node: OperationDefinitionNode, *_args: Any
     ) -> None:
-        if node.operation == OperationType.SUBSCRIPTION:
-            schema = self.context.schema
-            subscription_type = schema.subscription_type
-            if subscription_type:
-                operation_name = node.name.value if node.name else None
-                variable_values: Dict[str, Any] = {}
-                document = self.context.document
-                fragments: Dict[str, FragmentDefinitionNode] = {
-                    definition.name.value: definition
-                    for definition in document.definitions
-                    if isinstance(definition, FragmentDefinitionNode)
-                }
-                fields = collect_fields(
-                    schema,
-                    fragments,
-                    variable_values,
-                    subscription_type,
-                    node.selection_set,
-                    {},
-                    set(),
-                )
-                if len(fields) > 1:
-                    field_selection_lists = list(fields.values())
-                    extra_field_selection_lists = field_selection_lists[1:]
-                    extra_field_selection = [
-                        field
-                        for fields in extra_field_selection_lists
-                        for field in (
-                            fields
-                            if isinstance(fields, list)
-                            else [cast(FieldNode, fields)]
+        if node.operation != OperationType.SUBSCRIPTION:
```

Comment:
You just want to avoid one level of indentation, right? Again, this was a literal translation from GraphQL.js, but ok, we can do that.